guide/setup-integration-account.md

```diff
@@ -0,0 +1,17 @@
+# 1. Getting an Integration Account
+
+In order to get an Integration account and a fake test supplier:
```

Comment:
`fake` depends on the case, `test supplier` will just fit.

Reply:
Removing this. We're both being way to stubborn on this and spending too much time on it.